lib/database_connection.rb

```diff
@@ -0,0 +1,18 @@
+require 'pg'
+
+class DatabaseConnection
```

Comment:
Creating a DatabaseConnection class is a very nice idea to keep the code refactored.